### PR TITLE
Update README with implementations

### PR DESCRIPTION
**Pull request by kirubarajan**

### CODE REVIEW

**Review by n3a9 — COMMENTED** [review]
Do you think we can have two lists - one for data structures and one for algorithms?

File: README.md [hunk 1]
```diff
@@ -9,6 +9,12 @@ Why is this module named program.js? An inspiration for this project was compute
 
 ![Textbook](https://upload.wikimedia.org/wikipedia/en/9/90/Algorithms_%2B_Data_Structures.jpg)
 
+Current implementations:
+
+- Bubble sort
```
Comment (n3a9) [inline README.md]: Make it `- [x]` instead of just `-` so that it shows a check box.

**Review by n3a9 — CHANGES_REQUESTED** [review]
Change style of list.

File: README.md (hunk 1 above)
Comment (kirubarajan) [reply README.md]: I'll also add upcoming implementations with - []

Comment (kirubarajan) [reply README.md]: And I separated lists

**Review by n3a9 — APPROVED** [review]

**Pull request merged**